### DISCUSSION TITLE
Shortcut/URL to Pin focused app? `Pin [Application` opt in menubar

### DISCUSSION THREAD

**Discussion by al-ce** [rxhanson/RectanglePro-Community#223] [Q&A] [answered]
Is there a way to set the app to be pinned to be the focused/foremost app, like the menu bar option, via a keyboard shortcut or a URL? I tried looking for a URL in the discussions/issues but I might have missed it. Thanks!

**Comment by al-ce** [comment]
I ended up setting a Bartender hotkey for Rectangle Pro that opens up the menubar dropdown menu, then `p` + `enter`. I tried to get an AppleScript to do it but I couldn't find a way that didn't have this odd ~5 sec. delay after executing the click command.

**Comment by rxhanson** [comment, marked as answer]
As you noticed, currently there is no way within Rectangle Pro to do this. At the very least, I'll see about adding a URL in the next release, and potentially a keyboard shortcut for it.
